src/Chapter02/Chapter02.csproj

```diff
@@ -6,6 +6,9 @@
     <StartupObject>AddisonWesley.Michaelis.EssentialCSharp.Shared.Program</StartupObject> 
   </PropertyGroup>
   <Import Project="..\Versioning.targets" />
+  <ItemGroup>

```

Comment:
Looks like the project system got a little confused here, unless you actually want a code file to be in the project but not compiled.